File: src/AllKeyDecryption.java
import java.util.Scanner;

public class AllKeyDecryption {
    private Scanner sc = new Scanner(System.in);
    private String message;
    private String[] decryptedMessage = new String[26];

    public AllKeyDecryption(String message) {
        this.message = message;
    }

    public AllKeyDecryption() { }


    public String printDecryptedMessages() {
        String msg = "\n";
        for (int i = 0; i < 26; ++i) {
            //
            if(i < 10) msg += "key = " + i + " \t" + decryptedMessage[i] + "\n";
            else msg += "key = " + i + "\t" + decryptedMessage[i] + "\n";

        }
        return msg;
    }

    public void cipherMessage() {
        System.out.print("Enter a message: ");
        message = sc.nextLine();
    }

    public void decrypt() {
        String tempMessage = "";
        for (int i = 0; i < 26; ++i) {
            tempMessage = "";
            tempMessage = Decryption.getDecrypt(tempMessage, i, message);
            decryptedMessage[i] = tempMessage;
        }
    }



}

File: src/kotlin_caesar/AllKeyDecryption.kt
package kotlin_caesar
import java.util.*

class AllKeyDecryption {
    private val sc: Scanner = Scanner(System.`in`)
    private var message: String? = null
    private val decryptedMessage = arrayOfNulls<String>(26)

    constructor(message: String?) {
        this.message = message
    }

    constructor() {}

    fun printDecryptedMessages(): String {
        var msg = "\n"
        for (i in 0..25) { //
            msg += if (i < 10) "key = " + i + " \t" + decryptedMessage[i] + "\n" else "key = $i\t" + decryptedMessage[i] + "\n"
        }
        return msg
    }

    fun cipherMessage() {
        print("Enter a message: ")
        message = sc.nextLine()
    }

    fun decrypt() {
        var tempMessage = ""
        for (i in 0..25) {
            tempMessage = ""
            tempMessage = Decryption.getDecrypt(tempMessage, i, message)
            decryptedMessage[i] = tempMessage
        }
    }
}